tests/nightly/dist_sync_kvstore.py

```diff
@@ -279,10 +279,11 @@ def check_compr_random(kv, threshold, nworker):
                 assert_almost_equal(diff.asnumpy(), decompr)
 
     print ('worker '+str(my_rank)+' started with non compression tests')
-    check_default_keys(kv, my_rank, nworker)
     check_row_sparse_keys(kv, my_rank, nworker)
     check_row_sparse_keys_with_zeros(kv, my_rank, nworker)
     check_big_row_sparse_keys(kv, my_rank, nworker)
+    kv.set_merger(mx.merger.create('accumulate'))
```

Comment:
Is this only for dist kvstore? Can we also have unit test for "device" kvstore in https://github.com/apache/incubator-mxnet/blob/master/tests/python/gpu/test_kvstore_gpu.py 
and for "local" kvstore in https://github.com/apache/incubator-mxnet/tree/master/tests/python/unittest/test_kvstore.py ?